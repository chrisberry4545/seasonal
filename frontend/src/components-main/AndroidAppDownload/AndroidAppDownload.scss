@import './../../styles/settings';
@import './../../styles/tools';

.c-android-app-download {
  max-width: 200px;
  display: flex;
  align-items: center;
  position: relative;
  &:hover::after {
    content: '';
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    background-color: transparentize($background-color, 0.8);
  }
}

  .c-android-app-download__image {
    width: 100%;
  }
