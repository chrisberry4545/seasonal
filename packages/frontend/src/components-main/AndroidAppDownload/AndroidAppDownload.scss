@import './../../../node_modules/@chrisb-dev/seasonal-shared-frontend-components/dist/styles/settings';
@import './../../../node_modules/@chrisb-dev/seasonal-shared-frontend-components/dist/styles/tools';

.c-android-app-download {
  max-width: 200px;
  display: flex;
  align-items: center;
  position: relative;
  &:hover::after {
    content: '';
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
    background-color: transparentize($background-color, 0.8);
  }
}

  .c-android-app-download__image {
    width: 100%;
  }
