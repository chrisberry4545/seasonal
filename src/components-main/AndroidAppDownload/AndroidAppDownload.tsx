import React, { FC } from 'react';

import './AndroidAppDownload.scss';

export const AndroidAppDownload: FC<{}> = () => (
  <a className='c-android-app-download'
    href={
    'https://play.google.com/store/apps/details?id=com.chrisbdev.seasonal&p' +
      'campaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'
  }>
    <img className='c-android-app-download__image'
      alt='Get it on Google Play'
      src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png'/>
  </a>
);
