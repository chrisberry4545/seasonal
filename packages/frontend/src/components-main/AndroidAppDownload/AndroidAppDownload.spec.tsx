
import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { AndroidAppDownload } from './AndroidAppDownload';

describe('<AndroidAppDownload />', () => {
  let wrapper: ShallowWrapper;

  beforeEach(() =>
    wrapper = shallow(
      <AndroidAppDownload />
    )
  );

  test('renders correctly', () => expect(wrapper).toMatchSnapshot());

});
