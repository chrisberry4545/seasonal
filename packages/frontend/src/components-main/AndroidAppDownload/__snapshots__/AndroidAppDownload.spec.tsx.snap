// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AndroidAppDownload /> renders correctly 1`] = `
<a
  className="c-android-app-download"
  data-e2e="android-app-download"
  href="https://play.google.com/store/apps/details?id=com.chrisbdev.seasonal&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"
>
  <img
    alt="Get it on Google Play"
    className="c-android-app-download__image"
    src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png"
  />
</a>
`;
